src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/Owin/ResolveAppWrapper.cs

```diff
@@ -11,7 +11,7 @@ namespace NewRelic.Providers.Wrapper.Owin
     public class ResolveAppWrapper : IWrapper
     {
         public bool IsTransactionRequired => false;
-
+        private static bool _isNewRelicMiddlewareAdded;
```

Comment:
The same "safety" check concerns that apply to aspnetcore apply to owin as well. We need to see if this wrapper is ever executed concurrently.